controllers/potsController.js

```diff
@@ -0,0 +1,20 @@
+const potsService = require("../services/potsService");
+
+const potsDetails = async (req, res) => {
```

Comment:
```suggestion
const getPotInfo = async (req, res) => {
```
api를 구현함에 있어서 각각의 메소드 naming은 정확하게 정해진 규칙은 없지만, CRUD에 맞춰서 (create, get, update, delete) `동작과 자원의 조합`으로 지어주시면 모든 function에서 일관된 naming으로 작성할 수 있습니다. (마찬가지로 해당 데이터가 복수인지 단수인지도 구분되면 좋습니다)

단순 변수 naming일뿐이지만 통일성있고 명확하게 작성되면 직접 개발한 사람이 아닌 다른 사람이 보아도 의도와 흐름을 금방 파악할 수 있는 읽기 쉬운 코드가 됩니다 :)